backpack-compose/build.gradle.kts

```diff
@@ -29,6 +35,13 @@ android {
   composeOptions {
     kotlinCompilerExtensionVersion = rootProject.ext.get("compose_version") as String
   }
+
+  packagingOptions {
+    resources.excludes.add("**/attach_hotspot_windows.dll")
```

Comment:
What is this specific DLL for?

Reply:
It was as part of the coroutines test dependency i think

Reply:
right I see. I went through the coroutines issue https://github.com/Kotlin/kotlinx.coroutines/issues/2023